java/client/test/org/openqa/selenium/firefox/FireFoxPrintCommandTest.java

```diff
@@ -0,0 +1,76 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.firefox;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.After;
+import org.junit.Test;
+import org.junit.Before;
+
+import org.openqa.selenium.printoptions.PageSize;
+import org.openqa.selenium.printoptions.PrintOptions;
+import org.openqa.selenium.testing.JUnit4TestBase;
+
+public class FireFoxPrintCommandTest extends JUnit4TestBase {
```

Comment:
o_O Why is this not the same as the first test?

Reply:
Chrome needs to be headless, if not it throws an exception. Firefox doesn't need to headless.

Reply:
This still needs to be lifted up to the main test suite, and this class removed. If chrome can't handle this when not headless we should file a bug with the chrome team, ignore the test for chrome, and link to that bug in the ignore's comment.